gm-data-geonames/src/main/resources/reference.conf

```diff
@@ -0,0 +1,56 @@
+geomesa = {
+	sfts = {
```

Comment:
@vpipkt @elahrvivaz Do we have a preference for spaces over tabs in converter config?

Any ideas about enforcing silly style things like that?